cnr-ispc/.htaccess

```diff
@@ -3,5 +3,9 @@
 Header set Access-Control-Allow-Origin *
 Options +FollowSymLinks
 RewriteEngine on
- 
-RewriteRule ^ontology/branco$ https://dhilab-fi.github.io/branco/current/branco.owl [R=303,L]
\ No newline at end of file
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} html
```

Comment:
this is not a correct type, it should be text/html:
`RewriteCond %{HTTP_ACCEPT} text/html `

Reply:
It probably should be that.  But the `CondPattern` is a regex (+magic), so matching any `html` string is ok if that's what is desired.